### PR TITLE
rintrojs out for now, static tour

### PR DESCRIPTION
**Pull request by jvillacampa**
As the package author is trying to figure out the problem with rintrojs, I took that out and put an static version for now.

### CODE REVIEW

File: ui.R [hunk 1]
```diff
@@ -616,6 +616,68 @@ navbarMenu("Info", icon = icon("info-circle"),
...
+############## Tour of the tool----    
+###############################################.
+tabPanel("Tour of the tool", value = "tour",
+         mainPanel(width=11,
+                   fluidRow(p(h4("Welcome to the ScotPHO Profiles Tool"),
+                     h5("This interactive tool provides access to a range of public
+              health related indicators at different geographies including NHS boards, council areas and health and
+              social care partnerships.", style = "color:black;"),
+                     h5("There are different ways to navigate around the tool.", style = "color:black;"),
+                     h5("Different visualisations can be opened using the menu bar ('the blue strip') at the top of the screen.",
```
Comment (laurad05) [inline ui.R]: not sure 'the blue strip' needs to be in inverted commas. Or if it does then maybe make it the 'blue strip'

File: ui.R [hunk 2]
```diff
@@ -616,6 +616,68 @@ navbarMenu("Info", icon = icon("info-circle"),
...
+                        style = "color:black;"),
+                     img(src='introjs_tabset_panel.PNG',width=300),
+                     br(),
+                     h5("The 'Home' option in the menu bar will return to the profiles tool home page.",
```
Comment (laurad05) [inline ui.R]: would make home page all one word

File: ui.R [hunk 3]
```diff
@@ -616,6 +616,68 @@ navbarMenu("Info", icon = icon("info-circle"),
...
+                     img(src='tour_summary2.png', style = "vertical-align: middle; border-style: solid; border-color: black; border-width: 1px"),
+                     column(6, h5("While using dropdown menus mouse click within a dropdown menu and press backspace on your keyboard ('<-') then start typing a word to quickly find the options you are looking for",
+                                  style = "color:black;")),
+                     column(6, img(src='introjs_how_to_select.png')))),
+                   hr(),
+                   br(),
+                   fluidRow(column(8,
+                          p(h5("Throughout the tool look out for options in each window that provide",
+                               style = "color:black;"),
+                            tags$ul( tags$li("indicator defintions or help to interpret a visualisation,"),
```
Comment (laurad05) [inline ui.R]: typo in definitions (defintions)

File: ui.R [hunk 4]
```diff
@@ -616,6 +616,68 @@ navbarMenu("Info", icon = icon("info-circle"),
...
+                            tags$li("data download data options for individual charts,"),
```
Comment (laurad05) [inline ui.R]: Sentence is a bit funny having the word "data" in it twice - maybe just "data download options for individual charts"

File: ui.R [hunk 5]
```diff
@@ -616,6 +616,68 @@ navbarMenu("Info", icon = icon("info-circle"),
...
+                   hr(),
+                   br(),
+                   fluidRow(column(6,
+                          h5("The 'Data' window can be used to filter and download profiles data.",
+                             style = "color:black;")),
+                   column(6, img(src='tour_data1.PNG'))),
+                   hr(),
+                   br(),
+                   fluidRow(column(6,
+                          h5("The inequalities module allows exploration of deprivation affects a selection of indicators from the main profiles tool.",
```
Comment (laurad05) [inline ui.R]: "The inequalities module allows exploration of deprivation affects a selection of indicators from the main profiles tool."

Sentence is also a bit funny - should it say "The inequalities module allows **exploration of deprivation effects for a selection** of indicators from the main profiles tool." or something similar??

File: ui.R (hunk 3 above)
Comment (jvillacampa) [reply ui.R]: Done

File: ui.R (hunk 4 above)
Comment (jvillacampa) [reply ui.R]: Done

File: ui.R (hunk 1 above)
Comment (jvillacampa) [reply ui.R]: inverted commas out

File: ui.R (hunk 2 above)
Comment (jvillacampa) [reply ui.R]: done

File: ui.R (hunk 5 above)
Comment (jvillacampa) [reply ui.R]: done

**Pull request merged**